### DISCUSSION TITLE
Pass native `size` attribute to an `<input>` element

### DISCUSSION THREAD

**Discussion by bkwhite** [chakra-ui/chakra-ui#4239] [Q&A] [answered]
Reading the docs I don't see a way to pass a native `size` prop to an `<Input>` component.  

Is there a way to access the props being passed to the native `<input>` element? 

Thanks!

For reference: https://www.w3schools.com/tags/att_input_size.asp

**Comment by anubra266** [comment]
Just use width, But you can extend the theme, and define custom sizes

**Reply by bkwhite** [reply]
I've attempted to use `width`, it works okay but feels more brittle than I want.  Being able to use native `size` would let width be based on font-size.

**Comment by anubra266** [comment, marked as answer]
This should work 
```html
<chakra.input border="solid 1px gray" rounded="md" placeholder="Small" size="4" />
```

![image](https://user-images.githubusercontent.com/30869823/122304447-25228880-cefd-11eb-907f-990eb176ce57.png)

**Reply by bkwhite** [reply]
Perfect, thank you for your help! C:
